types/kaktana-react-lightweight-charts.d.ts

```diff
@@ -9,11 +12,13 @@ declare module "kaktana-react-lightweight-charts" {
         height?: number,
         options?: object,
         autoWidth?: boolean,
+				autoHeight?: boolean,
+				legend?: string;
```

Comment:
Why is there that much indentation? Please keep the project indentation of 2 spaces :)